### PR TITLE
Empty Total Routing RSI/MCI

### PR DESCRIPTION
**Pull request by dwyeradam**
Zero or empty routing for Total Retail Turnover and Employment questions for schemas:
1_0102, 1_0112, 1_0203, 1_0205, 1_0213 and 1_0215

Completion of https://github.com/ONSdigital/eq-survey-runner/pull/1509 that was merged early.

### CODE REVIEW

**Review by jonnyshaw89 — APPROVED** [review]

File: data/en/1_0112.json
Comment (jonnyshaw89) [inline data/en/1_0112.json:543]: This should route to `employees-breakdown`

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: data/en/1_0112.json
Comment (jonnyshaw89) [reply data/en/1_0112.json:543]: Im wrong. Ignore me

**Review by jonnyshaw89 — APPROVED** [review]

File: tests/functional/spec/features/default_value/default_value.spec.js
Comment (shahi645) [inline tests/functional/spec/features/default_value/default_value.spec.js]: Remove `@watch`.

**Review by shahi645 — CHANGES_REQUESTED** [review]

**Comment by shahi645** [comment]
We shouldn't merge this until SDX changes are merged in, otherwise they will just get transactions nacking. 
https://github.com/ONSdigital/sdx-transform-cs/pull/147

**Comment by jonnyshaw89** [comment]
@shahi645 not our problem.

**Comment by shahi645** [comment]
yeah but SDC is our service for the sake of not causing them an issue im sure we can hold off merging for like a day

**Comment by jonnyshaw89** [comment]
@shahi645 if we were going down that road then we shouldn't have merged this PR
https://github.com/ONSdigital/eq-survey-runner/pull/1512

If we are going to get these deployed for tomorrow then we need to get them in and tested asap

**Comment by shahi645** [comment]
i completely agree, but we cant test e2e without SDX changes, we can still test locally

**Review by shahi645 — APPROVED** [review]

**Pull request merged**
